api/tests/integration/infrastructure/repositories/campaign-participation-overview-repository_test.js

```diff
@@ -293,14 +325,17 @@ describe('Integration | Repository | Campaign Participation Overview', function
         });
       });
 
-      context('the filter is ARCHIVED', function () {
+      context('the filter is DISABLED', function () {
         it('returns participation where the campaign is archived', async function () {
```

Comment:
changer le it pour qu'il correspond mieux à ce qui est testé

Reply:
done